webview/src/plots/components/TooManyPlots.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { PropsWithChildren } from 'react'
+import { useSelector } from 'react-redux'
+import styles from './styles.module.scss'
+import { PlotsState } from '../store'
+import { MessageBand } from '../../shared/components/messageBand/MessageBand'
+import { Info } from '../../shared/components/icons'
+
+const PathHighlight: React.FC<PropsWithChildren> = ({ children }) => (
+  <span className={styles.pathHighlight}>{children}</span>
+)
+
+export const TooManyPlots: React.FC = () => {
+  const shouldShow = useSelector(
+    (state: PlotsState) => state.webview.shouldShowTooManyPlotsMessage
+  )
+
+  return shouldShow ? (
+    <MessageBand icon={Info} id="too-many-plots-message">
+      We are only showing 20 plots by default. To view other plots, you can
+      toggle their visibility in the sidebar. You can also group your plots by
+      adding a path prefix to their IDs (
```

Comment:
[C] In the `dvc.yaml` the ID for the plot can be a file or a string. The following are all valid (from demo project):

```
  - Loss:
      x: step
      y:
        training/plots/metrics/train/loss.tsv: loss
        training/plots/metrics/test/loss.tsv: loss
      y_label: loss
      title: "Loss"
  - Confusion matrix:
      template: confusion
      x: actual
      y:
        training/plots/sklearn/confusion_matrix.json: predicted
  - hist.csv:
      x: preds
      y: digit
      template: bar_horizontal
      title: Histogram of Predictions
  - probs.json:
      x: actual
      y: prob
      template: scatter
      title: Predicted Probabilities
  - training/plots/images
```

This means the instructions to group need to be updated slightly. In the case of the key being a file the user would need to group the underlying data. If it is just a string then it can be done in the suggested way.

Reply:
I changed the message. Does that look like complete instructions to you now @mattseddon ?